win32/devdoc/gballoc_ll_win32heap.md

```diff
@@ -207,3 +209,13 @@ MOCKABLE_FUNCTION(, void, gballoc_ll_print_stats);
 **SRS_GBALLOC_LL_WIN32HEAP_01_001: [** `gballoc_ll_print_stats` shall return without printing any statistics. **]**
 
 Note: printing of statistics is not implemented for `win32heap`.
+
+### gballoc_ll_set_decay
+
+```c
+MOCKABLE_FUNCTION(, int, gballoc_ll_set_decay, int64_t, decay_milliseconds);
+```
+
+`gballoc_ll_set_decay` does nothing and returns a non-zero value.
+
+**SRS_GBALLOC_LL_WIN32HEAP_28_001: [** `gballoc_ll_set_decay` shall do nothing and return a non-zero value. **]**

```

Comment:
>and return a non-zero value. [](http://example.com/codeflow?start=79&length=28)

I think it should succeed.

In the event where win32heap is used - I think that if this fails - the whole program might fail for no good reason.

Reply:
Done